scripts/packageComponent.js

```diff
@@ -123,8 +113,6 @@ const getNormalizedDATFileName = (datFileName) =>
   datFileName === 'debounce' ||
   datFileName === 'clean-urls' ||
   datFileName === 'messages' ||
-  datFileName === 'speedreader-updater' ||
-  datFileName === 'content-stylesheet' ||
```

Comment:
Just checking: `content-stylesheet` is related to speedreader?

Reply:
Yep! You can see it in another part of the diff, as well as [here](https://github.com/brave-experiments/SpeedReader/blob/master/data/content-stylesheet.css).